Error building app ?

./../../../.pub-cache/hosted/pub.dev/internet_connection_checker_plus-2.7.1/lib/src/internet_connection.dart:265:70: Error: A value of type 'StreamSubscription<ConnectivityResult>' can't be assigned to a variable of type 'StreamSubscription<List<ConnectivityResult>>?'.
 - 'StreamSubscription' is from 'dart:async'.
 - 'ConnectivityResult' is from 'package:connectivity_plus_platform_interface/src/enums.dart' ('../../../../.pub-cache/hosted/pub.dev/connectivity_plus_platform_interface-1.2.4/lib/src/enums.dart').
 - 'List' is from 'dart:core'.
    _connectivitySubscription = Connectivity().onConnectivityChanged.listen(
                                                                     ^
Target kernel_snapshot_program failed: Exception
2

FAILURE: Build failed with an exception.

--- a/lib/src/internet_connection.dart
+++ b/lib/src/internet_connection.dart
@@ -270,6 +270,7 @@ class InternetConnection {
           _maybeEmitStatusUpdate();
         }
       },
+      onError: (_, __) {},
     );
   }
 }

